src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -2359,7 +2362,8 @@ private boolean isFeedbackSessionFullyCompletedByInstructor(
      */
```

Comment:
Expand the code above my comment. The comment being referred to is outside the code you modified. It's the header comment of the method.